include/modem/nrf_modem_lib_trace.h

```diff
@@ -51,6 +51,20 @@ int nrf_modem_lib_trace_processing_done_wait(k_timeout_t timeout);
  */
 int nrf_modem_lib_trace_level_set(enum nrf_modem_lib_trace_level trace_level);
 
+#if defined(CONFIG_NRF_MODEM_LIB_TRACE_BITRATE) || defined(__DOXYGEN__)
+/** @brief Get the last measured average bitrate of the trace backend.
+ *
+ * This function returns the last measured average bitrate of the trace
+ * backend calculated over the @kconfig(CONFIG_NRF_MODEM_LIB_TRACE_BITRATE_PERIOD_MS)
+ * period.
+ *
+ * @param[out] value Rolling average bitrate of the trace backend
+ *
+ * @return Zero on success, non-zero otherwise.
+ */
+int nrf_modem_lib_trace_bitrate_get(double *value);
```

Comment:
Why is the `bitrate` a double? This should probably be an integer.

Reply:
Because it's an average

Reply:
Yeah, but it's an average over a number of bits.
There isn't anything smaller than a bit, so why are we interested in measuring `2.21908349832` bits instead `2` ?

Reply:
I think having this as a double would make sense if the transfer rate was measured in e.g. `kbps` (kilo bits per second). As this is the rate in bits per second, and we are talking about quite high numbers in general (up to ~100 000 bps) I agree with @lemrey that a resolution of whole bits per second is high enough.

Reply:
Ok, I removed floating point completely.